docs/clustering/multi-node-setup.rst

```diff
@@ -95,11 +95,12 @@ instructions.
 
        sh$ tar -xzf crate-*.tar.gz
 
-2. You should configure the :ref:`metadata gateway <metadata-gateway>` so that
-   CrateDB knows how to recover its state safely. Ideally, for a three-node
-   cluster, set `gateway.expected_data_nodes`_ to **3** and set
-   `gateway.recover_after_data_nodes`_ to **3**. You can specify these settings
-   in the `configuration`_ file of the unpacked directory.
+2. It is common to configure the :ref:`metadata gateway <metadata-gateway>` so 
+   that the cluster waits for all data nodes to be online before starting the
+   recovery of the shards. In this case let's set 
+   `gateway.expected_data_nodes`_ to **3** and 
+   `gateway.recover_after_data_nodes`_ also to **3**. You can specify these 
```

Comment:
Just fyi: in the case of crate-operator and therefore also Cloud `recover_after_data_nodes` is set to `floor(number_nodes / 2) + 1`